openshift/templates/app/pims-api/pims-api-deploy.yaml

```diff
@@ -146,6 +152,19 @@ parameters:
       Staging, Production
     required: true
     value: Production
+  - name: KEYCLOAK_SECRET
+    displayName: Keycloak Secret
+    description: The KeyCloak secret to use for authentication
+    required: false
+    value: ""
+  - name: KEYCLOAK_AUTHORITY
```

Comment:
I've sent you the config for Authority and Audience in RocketChat.

Reply:
Thanks, I have provided those in the template